linode/databasebackups/datasource_test.go

```diff
@@ -23,65 +20,28 @@ var (
 )
 
 func init() {
-	region, err := acceptance.GetRandomRegionWithCaps([]string{"Managed Databases"})
+	client, err := acceptance.GetClientForSweepers()
 	if err != nil {
-		log.Fatal(err)
-	}
-
-	testRegion = region
-}
-
-func TestFlattenBackup_MySQL(t *testing.T) {
-	currentTime := time.Now()
-
-	backup := linodego.MySQLDatabaseBackup{
-		ID:      123,
-		Label:   "cool",
-		Type:    "auto",
-		Created: &currentTime,
-	}
-
-	result := databasebackups.FlattenBackup(backup)
-	if result["id"] != backup.ID {
-		t.Fatal(cmp.Diff(result["id"], backup.ID))
-	}
-
-	if result["label"] != backup.Label {
-		t.Fatal(cmp.Diff(result["label"], backup.Label))
```

Comment:
Was this test case deleted for some reasons?

Reply:
I think we can make `parseMySQLBackup` and `parsePostgresSQLBackup` as independent functions and call them from `parseXXXXBackups` in the model file, and then bring these tests back to test those functions.

Reply:
As far as adding additional test cases, none of the parsing functions return anything so I'm not sure about the best way to test these.

Reply:
I think you can check the attributes in the model instance against the attributes in the linodego object

Reply:
I think this is optional because we didn't test model parsing in other data sources...

https://github.com/linode/terraform-provider-linode/actions/runs/5258218942
Let's wait for the action result, and you can decide whether to bring these tests back or not.

Reply:
The action result will fail due to an unresolved issue with polling. Still trying to figure it out.

Reply:
Solved!